src/catkin_pkg/changelog.py

```diff
@@ -53,6 +53,8 @@
 import pkg_resources
 import re
 
+from io import open
```

Comment:
oh interesting that didnt seem to be needed in #178 would read as binary (`rb`) work without this import ?